### PR TITLE
Feat/swarinfo 328 list swarm agent

### PR DESCRIPTION
**Pull request by xgroleau**

### CODE REVIEW

File: src/DefaultMessageOptions.cmake [hunk 1]
```diff
@@ -7,4 +7,5 @@ set_default(FUNCTION_ARGUMENT_NAME_SIZE 16)
 set_default(BUZZ_PAYLOAD_SIZE 16)
 
 set_default(BYTES_PAYLOAD_SIZE 200)
-set_default(NEIGHBORS_LIST_SIZE 50)
\ No newline at end of file
+set_default(NEIGHBORS_LIST_SIZE 50)
+set_default(AGENTS_LIST_SIZE 50)
```
Comment (t-constantin) [inline src/DefaultMessageOptions.cmake:11]: Shouldn't we make Neighbors list size = Agents list - 1?

**Review by t-constantin — APPROVED** [review]
Approved, but look into the comment I made just to be sure

**Review by g-lesssard — APPROVED** [review]
LGTM

File: src/DefaultMessageOptions.cmake (hunk 1 above)
Comment (xgroleau) [reply src/DefaultMessageOptions.cmake]: Fixed, and set to agent_list since it's the max, so it will take the same space in memory

**Pull request merged**